mods/ra/maps/top-o-the-world/scu36ea.lua

```diff
@@ -0,0 +1,361 @@
+			end)
+		end)
+	end)
+
+	Trigger.OnAnyKilled({ BridgeBarrel01, BridgeBarrel03 }, function()
+		if not BridgeEnd.IsDead then
+			BridgeEnd.Kill()
+		end
+	end)
+
+	Trigger.OnAnyKilled(AlliedSquad04, function()
+		if BridgeIsIntact then
+			local tanks = Reinforcements.Reinforce(greece, AlliedTanksReinforcement, { AlliedHuntingPartySpawn.Location, AlliedHuntingPartyWP01.Location }, 0, function(units)
+				AlliedGroundPatrols(units)
+			end)
+			Trigger.OnAllKilled(tanks, function()
+				if BridgeIsIntact then
+					Reinforcements.Reinforce(greece, AlliedTanksReinforcement, { AlliedHuntingPartySpawn.Location, AlliedHuntingPartyWP01.Location }, 0, function(units)
+					AlliedGroundPatrols(units)
+					end)
+				end
+			end)
+		end
+	end)
+
+	Trigger.OnAllKilled(AlliedSquad04, function()
+		if BridgeIsIntact then
+			local tanks = Reinforcements.Reinforce(greece, AlliedTanksReinforcement, { AlliedHuntingPartySpawn.Location, AlliedHuntingPartyWP01.Location }, 0, function(units)
+				AlliedGroundPatrols(units)
+			end)
+			Trigger.OnAllKilled(tanks, function()
+				if BridgeIsIntact then
+					Reinforcements.Reinforce(greece, AlliedTanksReinforcement, { AlliedHuntingPartySpawn.Location, AlliedHuntingPartyWP01.Location }, 0, function(units)
+					AlliedGroundPatrols(units)
+					end)
+				end
+			end)
+		end
+	end)
+
+--Units Death Setup
+	Trigger.AfterDelay(DateTime.Seconds(660), function()
+		Utils.Do(USSRDie01, function(actor)
+			if not actor.IsDead then actor.Kill() end
+		end)
+	end)
+
+	Trigger.AfterDelay(DateTime.Seconds(744), function()
+		Utils.Do(USSRDie02, function(actor)
+			if not actor.IsDead then actor.Kill() end
+		end)
+	end)
+
+	Trigger.AfterDelay(DateTime.Seconds(1122), function()
+		if not USSRHTank03.IsDead then USSRHTank03.Kill() end
+	end)
+
+	Trigger.AfterDelay(DateTime.Seconds(1230), function()
+		Utils.Do(USSRDie03, function(actor)
+			if not actor.IsDead then actor.Kill() end
+		end)
+	end)
+
+	Trigger.AfterDelay(DateTime.Seconds(1338), function()
+		Utils.Do(USSRDie04, function(actor)
+			if not actor.IsDead then actor.Kill() end
+		end)
+	end)
+
+	Trigger.AfterDelay(DateTime.Seconds(1416), function()
+		Utils.Do(USSRDie05, function(actor)
+			if not actor.IsDead then actor.Kill() end
+		end)
+	end)
+
+	Trigger.AfterDelay(DateTime.Seconds(1668), function()
+		if not USSRV201.IsDead then USSRV201.Kill() end
+	end)
+
+	Trigger.AfterDelay(DateTime.Seconds(1746), function()
+		Utils.Do(USSRDie06, function(actor)
+			if not actor.IsDead then actor.Kill() end
+		end)
+	end)
+
+	Trigger.AfterDelay(DateTime.Seconds(2034), function()
+		if not USSRMTank02.IsDead then USSRMTank02.Kill() end
+	end)
+
+	Trigger.AfterDelay(DateTime.Seconds(2142), function()
+		if not USSRMTank01.IsDead then USSRMTank01.Kill() end
+	end)
+
+	Trigger.AfterDelay(DateTime.Minutes(36), function()
+		if not ObjectiveTruck01.IsDead then ObjectiveTruck01.Kill() end
+	end)
+end
+
+Tick = function()
+
+-- Timer Setup
```

Comment:
This would have been the perfect use case for my time limit PR. Oh well.

Reply:
Do you want to update this in your PR?

Reply:
I could rebase it now and do it, yes.